MAUI/ListView/working-with-sflistview.md

```diff
@@ -0,0 +1,769 @@
+---
+layout: post
+title: Working with ListView in .NET MAUI ListView control | Syncfusion
+description: Learn here all about Working with ListView support in Syncfusion .NET MAUI ListView control and more.
+platform: MAUI
+control: SfListView
+documentation: ug
+---
+
+# Working with ListView in .NET MAUI ListView (SfListView)
+
+## Interacting with ListView Items
+
+### Loaded event
+
+The [Loaded](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_Loaded) event is raised when the `SfListView` is loading in view for the first time.
+
+{% tabs %}
+{% highlight c# %}
+
+listView.Loaded += ListView_Loaded;
+
+private void ListView_Loaded(object sender, Syncfusion.Maui.ListView.ListViewLoadedEventArgs e)
+{
+   listView.SelectedItems.Add(viewModel.Customers[2]);
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+The `Loaded` event used for the following use cases:

```

Comment:
The `Loaded` event is used for the following use cases:

Reply:
I have modified the line as per the concerns provided